infrastructure/lambda_differentiate.tf

```diff
@@ -0,0 +1,29 @@
+## function
+
+resource "aws_lambda_function" "differentiate_input_images" {
+  filename      = "lambda_scr_build/differentiate.zip"
+  function_name = "differentiate_input_images"
+  role          = aws_iam_role.differentiate_lambda_role.arn
+  timeout       = 25
```

Comment:
is 25 seconds really enough. I would increase it a little bit. timeout just means that your lambda function gets terminated when it takes longer. So it is not a real problem to set it to 60 seconds.